### DISCUSSION TITLE
any Plans to get rid of less?

### DISCUSSION THREAD

**Discussion by momesana** [ant-design/ant-design-mobile#5821] [Q&A] [answered]
Antd has finally moved on from less to css-in-js. Will antd mobile follow suit? Currently the styling is still largely based on less.js which is a pain to work with in many regards (especially supporting dynamic themes).

**Comment by zombieJ** [comment, marked as answer]
Since antd-mobile v5 release in one year, it may not quick do a breaking change. But yes, I've take this in consider.
